<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo QListWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Element Baru</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtElement">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRightAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLeft">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLeftAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
